src/components/v2/chartDeploymentHistory/chartDeploymentHistory.service.ts

```diff
@@ -40,27 +40,25 @@ interface RollbackReleaseResponse extends ResponseType {
 }
 
 export const getDeploymentHistory = (appId: string, isExternal: boolean): Promise<ChartDeploymentHistoryResponse> => {
-    const url = isExternal
-        ? `${Routes.HELM_RELEASE_DEPLOYMENT_HISTORY_API}?appId=${appId}`
-        : `${Routes.APP_RELEASE_DEPLOYMENT_HISTORY_API}?installedAppId=${appId}`
-    return get(url)
+    return get(
+        `${Routes.HELM_RELEASE_DEPLOYMENT_HISTORY_API}?${
+            isExternal ? `appId=${appId}&installedAppId=` : `installedAppId=${appId}&appId=`
```

Comment:
installedAppId should be nonmandatory in case of EA. and appId should not be mandatory in case of devtron chart.

(Please get in touch with vikram for this)